File: src/main/java/org/jbake/template/RenderingException.java
package org.jbake.template;

/**
 * Thrown if rendering of a document failed.
 *
 * @author Cédric Champeau
 */
public class RenderingException extends Exception {
    public RenderingException(final Throwable cause) {
        super(cause);
    }

    public RenderingException(final String message, final Throwable cause) {
        super(message, cause);
    }

    public RenderingException(final String message) {
        super(message);
    }
}

File: jbake-core/src/main/java/org/jbake/template/RenderingException.kt
package org.jbake.template

/**
 * Thrown if rendering of a document failed.
 *
 * @author Cédric Champeau
 */
open class RenderingException : Exception {
    constructor(cause: Throwable?) : super(cause)

    constructor(message: String?, cause: Throwable?) : super(message, cause)

    constructor(message: String?) : super(message)
}
